crates/trade/src/cfd.rs

```diff
@@ -54,3 +58,61 @@ pub fn calculate_short_liquidation_price(leverage: Decimal, price: Decimal) -> D
 
     price * leverage / (leverage - Decimal::ONE)
 }
+
+// TODO: This was copied from ItchySats and adapted; we need tests for this!
+/// Compute the payout for the given CFD parameters at a particular `closing_price`.
```

Comment:
Can you update the doc please? It contains things which are not relevant anymore.